src/properties/transform.rs

```diff
@@ -1457,23 +1457,25 @@ impl ToCss for Perspective {
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "jsonschema", derive(schemars::JsonSchema))]
 #[cfg_attr(feature = "into_owned", derive(static_self::IntoOwned))]
-pub struct Translate {
-  /// The x translation.
-  pub x: LengthPercentage,
-  /// The y translation.
-  pub y: LengthPercentage,
-  /// The z translation.
-  pub z: Length,
+pub enum Translate {
+  /// The "none" keyword.
+  None,
+
+  /// The x, y, and z translations.
+  XYZ {
```

Comment:
We could use a better name, but I don't hate it. I also noticed there are some structs like this in the `colors.rs` file. E.g.: `XYZd50`